### PR TITLE
Makes sound play throught the entire session

### PR DESCRIPTION
**Pull request by lucasalencar**

### CODE REVIEW

File: app/views/meditation_sessions/_audio.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<%= audio_tag('bell.mp3') %>
+<audio id="bell" hidden='true' controls>
+  <source src="<%= audio_path('bell.mp3') %>"></source>
+</audio>
```
Comment (marcelinol) [inline app/views/meditation_sessions/_audio.html.erb]: remove it

File: app/assets/javascripts/audio_playback.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  'use strict';
+
+  var bell = $('#silence-bell')[0];
+  var bellInterval;
+
+  $(window).on('start-session', function() {
+    bellInterval = setInterval(function() {
+      bell.currentTime = 0;
+      bell.play();
+    }, 1000);
```
Comment (marcelinol) [inline app/assets/javascripts/audio_playback.js]: lets see this on mobile

**Comment by marcelinol** [comment]
:ship:

File: app/assets/javascripts/audio_playback.js (hunk 2 above)
Comment (lucasalencar) [inline app/assets/javascripts/audio_playback.js]: agreed!

**Pull request merged**